pydrive2/test/README.rst

```diff
@@ -25,11 +27,13 @@ Run tests locally
       tests", and click **Create**. Select **Continue** at the next **Service
       account permissions** page, click at **+ CREATE KEY**, select **P12** and
       **Create**. Save generated **.p12** key file at your local disk.
-    - Edit **/pydrive2/test/settings/local/default.yaml** by replacing email
-      **your-service-account-email** with your new created service account
+    - Edit files **/pydrive2/test/settings/local/default.yaml** and
+      **/pydrive2/test/settings/local/test_oauth_test_06.yaml** by replacing
+      **your-service-account-email** with email of your new created service account
       and **your-file-path.p12** with path to the downloaded **.p12** key file.
```

Comment:
path to p12 - relative to the root or absolute

Reply:
it can be both

Reply:
I meant that we need to specify this - that it can be absolute or if it's relative that it must be relative to root

better to ask people to copy the file to a determined place

Reply:
that is addressed in https://github.com/iterative/PyDrive2/pull/15/commits/6c113695fc5d3146da24102f6c3a968d3ddcdcf0